keyboards/keebio/iris/iris.h

```diff
@@ -10,6 +10,8 @@
     #include "rev3.h"
 #elif KEYBOARD_keebio_iris_rev4
     #include "rev4.h"
+#elif KEYBOARD_keebio_iris_rev5
```

Comment:
These should probably be:
```suggestion
#elif defined(KEYBOARD_keebio_iris_rev5)
```

Can't tag the previous versions though

Reply:
Fixed for all versions now